velox/vector/LazyVector.cpp

```diff
@@ -134,10 +134,11 @@ void LazyVector::ensureLoadedRows(
     baseRows.resize(0);
     baseRows.resize(lazyVector->size(), false);
     rows.applyToSelected([&](auto row) {
-      if (!decoded.isNullAt(row)) {
-        baseRows.setValid(decoded.index(row), true);
-      }
+      baseRows.setValid(decoded.index(row), true);
     });
+    // Manually mark the last item to be selected to ensure end
+    // being no less than the expected size.
+    baseRows.setValid(lazyVector->size() - 1, true);
```

Comment:
This is a tmp tryout to make `baseRows.end()` no less than `lazyVector->size()`, and can be not appropriate.